SimG4CMS/Calo/src/HFShower.cc

```diff
@@ -341,6 +341,7 @@ std::vector<HFShower::Hit> HFShower::getHits(const G4Step *aStep, bool forLibrar
   std::vector<double> momz = cherenkov_->getMom();
 
   for (int i = 0; i < npe; ++i) {
+    hit.depth = 0;

```

Comment:
I would have (naively) thought that here the `depth` calculated above should have been assigned instead of 0...
Just fpr check: is `hit.depth = 0` the correct assignment here?

Reply:
you are right @perrotta , at https://github.com/cms-sw/cmssw/blob/99e00f24e2a0423766f85da9b55d469c1f35a2b2/SimG4CMS/Calo/src/HFShower.cc#L240 also does the same. I have updated it to use the depth calculated above